app/src/app/content-plugin/ui/application.scss

```diff
@@ -2,6 +2,7 @@
 @import 'mixins';
 @import 'theme';
 @import 'variables/font-family';
+@import 'custom-theme';
```

Comment:
`theme` already contains import for `custom-theme`, you don't need it here

Reply:
Fixed